python/grass/jupyter/display.py

```diff
@@ -0,0 +1,48 @@
+# MODULE:    grass.jupyter.display
+#
+# AUTHOR(S): Caitlin Haedrich <caitlin DOT haedrich AT gmail>
+#
+# PURPOSE:   This module contains functions for non-interactive display
+#            in Jupyter Notebooks
+#
+# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+#
+#           This program is free software under the GNU General Public
+#           License (>=v2). Read the file COPYING that comes with GRASS
+#           for details.
+
+import os
+import grass.script as gs
+
+
+class GrassRenderer:
+    """
+    The grassRenderer class creates and displays GRASS maps in Jupyter
+    Notebooks.
+    """
```

Comment:
GrassRenderer and ideally one line including the triple quotes.

Reply:
For listing of data also:
https://grass.osgeo.org/grass79/manuals/libpython/script.html?highlight=list#script.core.list_strings
could be used...